ENIGMAsystem/SHELL/ENIGMAengine.cbp

```diff
@@ -946,15 +946,15 @@
 		<Unit filename="Universal_System/IMGloading.h" />
```

Comment:
Can't we delete this file?

Reply:
I've removed it because it no longer exists: https://github.com/enigma-dev/enigma-dev/pull/1157/commits/246d20a8b99b6d67e2db6d7a908e16f37cb657ce

However, it looks like `GLESspriteadd.cpp` still refers to it. I believe it was actually renamed to `image_formats.h` which we still have and appears elsewhere in this manifest.